web/yo/app/scripts/services/mainutils.js

```diff
@@ -703,6 +822,11 @@ angular.module('oncokbApp')
             therapyStrToArr: therapyStrToArr,
             getDrugNameByUuids: getDrugNameByUuids,
             drugUuidtoName: drugUuidtoName,
-            getKeysWithoutFirebasePrefix: getKeysWithoutFirebasePrefix
+            getKeysWithoutFirebasePrefix: getKeysWithoutFirebasePrefix,
```

Comment:
You need to consider whether you want to export all following methods. If they are not potentially used, they should not be exported. And since they all in mainUtils, they need to be generic enough in order to have the name like `changeMapWhenAdd`. You could also consider leaving these methods to DrugMapUtils for less strict naming convention.